simpleprophet/data.py

```diff
@@ -0,0 +1,171 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pandas as pd
+
+
+_kpi_queries = {
```

Comment:
I'd expect constants like these to be all caps like KPI_QUERIES, but also respect a leading underscore for "private" implementation details.

Reply:
Fixed!